[exporter/loadbalancingexporter] update loadbalancing exporter readme (#18219)
open-telemetry/opentelemetry-collector-contrib: exporter/loadbalancingexporter/README.md

| Supported pipeline types | traces, logs |
| Distributions            | [contrib]    |

> This is an exporter that will consistently export spans and logs depending on the `routing_key` configured. If no `routing_key` is configured, the default routing mechanism in `traceID` i.e; this exporter has an internal algorithm that in a distributed environment spans belonging to the same `traceID` and `service.name` will be sent to the same backend.

Reviewer: ```suggestion
This is an exporter that will consistently export spans and logs depending on the `routing_key` configured. If no `routing_key` is configured, the default routing mechanism in `traceID` . This means that spans belonging to the same `traceID` (or `service.name`, when `service` is used as the `routing_key`) will be sent to the same backend.
```
